enterprise/reporting/error_extractor.go

```diff
@@ -83,7 +83,7 @@ func (ext *ExtractorHandle) getSimpleMessage(jsonStr string) string {
 				return strings.Split(erResStr, "\n")[0]
 			}
 			return ""
-		case responseKey:
+		case responseKey, "error":
```

Comment:
We can create a constant for error string?

Reply:
Done